docs/plugins/alerting.asciidoc

```diff
@@ -0,0 +1,18 @@
+[[alerting]]
+== Alerting Plugins
+
+Alerting plugins allow Elasticsearch to monitor indices and to trigger alerts when thresholds are breached.
+
+[float]
+=== Core alerting plugins
+
+The core alerting plugins are:
+
+link:/products/watcher[Watcher]::
+
+Watcher is the alerting and notification product for Elasticsearch that lets
+you take action based on changes in your data. It is designed around the
+principle that if you can query something in Elasticsearch, you can alert on
+it. Simply define a condition, a schedule, and the actions to take, and
+Watcher will do the rest.
```

Comment:
I'm not sure it's necessary to call it out at this level.  My feeling is that we either update all these links with their licenses (assuming we can find them and keep them up to date), or we put the onus on the plugin user to do their homework to make sure the version of the plugin/client/integration has a license they are comfortable with. The later approach is much easier for us to accurately maintain.